cmd/ipfs/main.go

```diff
@@ -47,6 +47,44 @@ const (
 	heapProfile        = "ipfs.memprof"
 )
 
+type cmdInvocation struct {
```

Comment:
Looks like some code that got deleted in another PR is getting reintroduced here. cc @magik6k

Reply:
Fixed, it was legacy code accidentally put up with second commit.